services/parser/parser.go

```diff
@@ -42,7 +42,12 @@ type (
 	}
 )
 
-const MinTxsBatchLimit = 500
+const (
```

Comment:
can we move this to config?

Reply:
any sense? it is not changable for years?

Reply:
For consistency, most params we specify in config